genesis-io/nexus_instinct
# [NXI-2] Add user authentication

#### Relevant Issues
https://github.com/genesis-io/nexus_instinct/issues/2
#### Description of PR
This PR introduces and login/signup flow but doesn't include actual api calls just yet. It also introduces architecture setup for styling components, mobx store and http based calls. We should use the following patterns implemented in this pr going forward.

#### Manual test procedure
* pull branch down locally
* yarn install
* navigate to localhost:3000/login and verify that the page looks good in mobile, tablet and desktop views.
* repeat the same steps from previous bullet with localhost:3000/signup
